jupyterlab_gallery/manager.py

```diff
@@ -81,6 +81,12 @@ def __init__(self, *args, **kwargs):
         config=True,
     )
 
+    clone_timeout = Int(
+        help="Timeout for cloning a repository",
+        default_value=60,
```

Comment:
I guess 1 minute will happen quite often when cloning larger repos, especially during tutorials on conferences with overloaded WiFi networks. I think we should go for something much higher for a default, maybe 15 minutes? Not sure.

```suggestion
        default_value=15 * 60,
```